keras/random/seed_generator.py

```diff
@@ -89,6 +89,13 @@ def next(self, ordered=True):
             self.state.assign((seed_state + 1) * 5387 % 933199)
         return new_seed_value
 
+    def get_config(self):
+        return {"seed": self._initial_seed, "backend": self.backend}
```

Comment:
You can drop the `backend` key. It's not serializable anyway

Reply:
done!